Dockerfile

```diff
@@ -15,60 +15,34 @@
 # limitations under the License.
 #
 
-######################################################################
-# PY stage that simply does a pip install on our requirements
-######################################################################
-ARG PY_VER=3.8.13-slim
-FROM python:${PY_VER} AS superset-py
-
-RUN mkdir /app \
-        && apt-get update -y \
-        && apt-get install -y --no-install-recommends \
-            build-essential \
-            default-libmysqlclient-dev \
-            libpq-dev \
-            libsasl2-dev \
-            libecpg-dev \
-        && rm -rf /var/lib/apt/lists/*
-
-# First, we just wanna install requirements, which will allow us to utilize the cache
-# in order to only build if and only if requirements change
-COPY ./requirements/*.txt  /app/requirements/
-COPY setup.py MANIFEST.in README.md /app/
-COPY superset-frontend/package.json /app/superset-frontend/
-RUN cd /app \
-    && mkdir -p superset/static \
-    && touch superset/static/version_info.json \
-    && pip install --no-cache -r requirements/local.txt
-
-
 ######################################################################
 # Node stage to deal with static asset construction
 ######################################################################
+ARG PY_VER=3.8.13-slim
```

Comment:
The section is the Node image so I think `PY_VER` is useless here.

Reply:
This is some not well documented in docker document.  in a multistage dockerfile, ARG used in FROM need to be added before the first FROM. 

<img width="1131" alt="image" src="https://user-images.githubusercontent.com/1592014/194346601-22c56a35-26c2-45cc-9dc7-a3c608d4a0f2.png">